### PR TITLE
Added Arabic Translation for all Intro-To-Storybook tutorials (React)

### PR DESCRIPTION
**Pull request by eihabkhan**
Translated all React-based "Intro-To-Storybook" tutorials into Arabic:

- Get started
- Simple component
- Composite component
- Data
- Screens
- Deploy
- Testing
- Addons
- Conclusion
- Contribute

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@eihabkhan wow, this is really awesome, thank you very much for the time and effort you put into this. We really appreciate it. Going to take a closer look at it during my weekend and let you know of any changes that need addressing so that we can move on to the proofread. Sounds good?

Stay safe

**Comment by eihabkhan** [comment]
Sounds perfect!

File: content/intro-to-storybook/react/ar/get-started.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+```shell
+git add .
+```
+
+ثم أخيرا:
+
+```shell
+git commit -m "first commit"
+```
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react/ar/get-started.md:75]: @eihabkhan if you could leave the code snippet comments in English throughout the tutorial we'd appreciate it. We're aware that translating the snippets brings the translation closer to home. But we're trying to work on a good solution for this situation.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@eihabkhan I ran through the tutorial, and we're almost in good shape to get a contributor to look into it and proofread it. I have left a couple of items that need to be addressed, and also I want to ask you something.

I was checking this and it seems that the snippets are working to some extent. But some of them are completely inverted and they're not properly formatted and displayed. With this, I would like to ask if you are willing to leave the tutorial's text wrapped into a div with the RTL  to properly format it to your native language and leave the snippets as they are. I'm aware that it's a bit cumbersome and I might be asking a bit much, but this is a bit of a temporary solution until we track down the culprit and we're able to fix the problem.

Let me know if you're ok with it and we'll go from there.

Stay safe

**Comment by eihabkhan** [comment]
Noted!
@jonniebigodes thank you for your feedback, I'll work on it over the weekend.

File: content/intro-to-storybook/react/ar/composite-component.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
+---
+title: 'جمّع مكون مركب'
+tocTitle: 'مكون مركب'
+description: 'جمّع مكون مركب من مكونات بسيطة'
+commit: 'f9b2cfb'
+---
+
+<div style="direction: rtl">
+
+بنينا في أخر فصل مكوننا الأول; هذا الفصل يوسع مع تعلمناه لبناء قائمة مهام, قائمة بها مجموعة مهام. لندمج مكونات معا و نرى ما سيحصل عندما يتم إضافة متسوى جديد من التعقيد.
```
Comment (merodiro) [inline content/intro-to-storybook/react/ar/composite-component.md:10]:
```suggestion
بنينا في أخر فصل مكوننا الأول; هذا الفصل يوسع مع تعلمناه لبناء قائمة مهام, قائمة بها مجموعة مهام. لندمج مكونات معا و نرى ما سيحصل عندما يتم إضافة مستوى جديد من التعقيد.
```

File: content/intro-to-storybook/react/ar/composite-component.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+نص الوصف المٌضاف يٌنتج الواجهة التالية:
+
+<video autoPlay muted playsInline loop>
+  <source
+    src="/intro-to-storybook/finished-tasklist-states-6-0.mp4"
+    type="video/mp4"
+  />
+</video>
+
+لاحظ موضع العنصر المثبت في اللائحة. نريد إظهار العنصر المثبت في أعلا الائحة لإعطائه أولوية لمستخدمينا.
```
Comment (merodiro) [inline content/intro-to-storybook/react/ar/composite-component.md:208]:
```suggestion
لاحظ موضع العنصر المثبت في اللائحة. نريد إظهار العنصر المثبت في أعلى الائحة لإعطائه أولوية لمستخدمينا.
```

File: content/intro-to-storybook/react/ar/composite-component.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+
+## إختبار مميكن
+
+في الفصل السابق تعلمنا كيفية إجراء إختبار لمحة على الستوريز خاصتنا بإستخدام Storyshots. مع `Task` لا يوجد الكثير من التقيد لإختبار أن الإظهار يجري بشكل صحيح. بما أن `TaskList` تضيف طبق أخرى من التعقيد نريد التأكد من أن بعض المدخلات تنتج إخراجات محددة بطريقة تتيح الإختبار المميكن. للقيام بذلك, سننشئئ إختبارات وحدة بإستخدام [مكتبة رياكت للإختبار](https://testing-library.com/docs/react-testing-library/intro) و [@storybook/testing-react](https://storybook.js.org/addons/@storybook/testing-react).
+
+![شعار مكتبة الإختبار](/intro-to-storybook/testinglibrary-image.jpeg)
+
+### إختبارات الوحدة و مكتبة إختبار رياكت
+
+ستوريز الخاصة بستوريبوك و الإختبارات اليدوية و إختبارات اللمحة تساعد كثيرا في تجنب أخطاء الواجهات. إذا غطت الستوريز أنواع مختلفة من حالات إستخدان المكون, وإستخدمنا الأدوات التي تضمن أن شخص سيرى التغيير الذي يطرأ على ستوري, تكون إحتمالية وقوع الأخطاء قليلة بشكل ملحوظ.
```
Comment (merodiro) [inline content/intro-to-storybook/react/ar/composite-component.md:251]:
```suggestion
ستوريز الخاصة بستوريبوك و الإختبارات اليدوية و إختبارات اللمحة تساعد كثيرا في تجنب أخطاء الواجهات. إذا غطت الستوريز أنواع مختلفة من حالات إستخدام المكون, وإستخدمنا الأدوات التي تضمن أن شخص سيرى التغيير الذي يطرأ على ستوري, تكون إحتمالية وقوع الأخطاء قليلة بشكل ملحوظ.
```

File: content/intro-to-storybook/react/ar/screen.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+tocTitle: 'واجهات'
+description: 'أنشئ واجهة من المكونات'
+commit: '46f29e3'
+---
+
+<div style="direction: rtl">
+
+ركزنا على بناء الواجهات من الأسفل لأعلى; عن طريق البدأ بشكل صغير ثم إضافة التعقيدات. هذا مكننا من تطوير كل مكون في منعزل, تحديد بياناته, و التلاعب به في ستوريبوك. كل ذلك دون الحاجة لخادم او بناء واجهات.
+
+في هذا الفصل سنستمر في إضافة متوى تعقيدي بدمج المكونات في واجهة و تطوير تلك الواجهة في ستوريبوك
```
Comment (merodiro) [inline content/intro-to-storybook/react/ar/screen.md:12]:
```suggestion
في هذا الفصل سنستمر في إضافة مستوى تعقيدي بدمج المكونات في واجهة و تطوير تلك الواجهة في ستوريبوك
```

Comment (eihabkhan) [reply content/intro-to-storybook/react/ar/screen.md]: متشكر جدا يا عمر 👍

File: content/intro-to-storybook/react/ar/deploy.md [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+
+تنفيذ `yarn build-storybook` سينتج ستوريبوك ثابت في وجهة `storybook-static`, و التي يمكنها بعد ذلك الإطلاق على أي خدمة إستضافة مواقع ثابتة.
+
+## نشر ستوريبوك
+
+هذا الدرس يستخدم <a href="https://www.chromatic.com/">كروماتك</a>, خدمة نشر مجانية صنعت من مراقبي ستوريبوك. تسمح لنا بإطلاق و إستضافة ستوريبوك خاصتنا بشكل أمن في السحابة.
+
+### قم بإعداد مستودع في Github
+
+قبل أن نبدأ, يحتاج الكود المحلي ان يتزامن مع خدمة التحكم في الإصدار عن بعد. عندما تم تهيئة مشروعنا في [فصل البدأ](/intro-to-storybook/react/en/get-started/), قمنا بتهيئة مستودع محلي. في هذه اللحظة ليدنا بالفعل مجموعة من التنفيذات (commits) التي يمكننا دفعها إلى مستودع عن بعد.
```
Comment (merodiro) [inline content/intro-to-storybook/react/ar/deploy.md:24]:
```suggestion
قبل أن نبدأ, يحتاج الكود المحلي ان يتزامن مع خدمة التحكم في الإصدار عن بعد. عندما تم تهيئة مشروعنا في [فصل البدأ](/intro-to-storybook/react/en/get-started/), قمنا بتهيئة مستودع محلي. في هذه اللحظة لدينا بالفعل مجموعة من التنفيذات (commits) التي يمكننا دفعها إلى مستودع عن بعد.
```

**Comment by jonniebigodes** [comment]
@eihabkhan great to see that this is getting some traction and someone is already proofreading it. That's great. Just one small note on this. If you could factor in #504 I'd appreciate it.

**Pull request closed without merging**